pytest_rerunfailures.py

```diff
@@ -167,20 +176,38 @@ def get_reruns_delay(item):
         if "reruns_delay" in rerun_marker.kwargs:
             delay = rerun_marker.kwargs["reruns_delay"]
         elif len(rerun_marker.args) > 1:
-            # check for arguments
+            # check for argument.
             delay = rerun_marker.args[1]
         else:
             delay = 0
+
+        if "reruns_delay_backoff" in rerun_marker.kwargs:
+            delay_backoff = rerun_marker.kwargs["reruns_delay_backoff"]
+        elif len(rerun_marker.args) > 2:
+            # check for argument.
+            delay_backoff = rerun_marker.args[2]
```

Comment:
It looks like you're expecting a parameter to `--reruns-delay-backoff` here? But elsewhere you're just checking for the presence of that flag.

Reply:
In this case the argument is True or False

Reply:
So checking as a flag or True is the same